CHANGELOG.md

```diff
@@ -24,6 +24,25 @@ CHANGELOG
 Swift 5.0
 ---------
 
+* [SE-0235][]:
+
+  The standard library now contains a `Result` type for manually propagating errors.
+  
+  ```swift
+  enum Result<Success, Failure: Error> {
+      case success(Success)
+      case failure(Failure)
+  }
+  ```
+  
+  This type serves a complementary role to that of `throws`. `Result` can be used 
+  in situations where automatic error propagation or the ceremony of `try` / `catch` 
+  are undesirable, such as in asynchronous code or in accumulating the results of 
+  successive error-producing operations.
```

Comment:
How about this? I'd like to avoid the passive voice and calling `try`-`catch` a "ceremony":

> This type serves a complementary role to that of throwing functions and initializers. Use `Result` in situations where automatic error propagation or `try`-`catch` blocks are undesirable, such as in asynchronous code or when accumulating the results of successive error-producing operations.